.github/workflows/release.yml

```diff
@@ -4,7 +4,7 @@ on:
     branches: ["main"]
   pull_request:
     branches: ["main"]
-    types: [opened, synchronize, labled, unlabled]
+    types: [opened, synchronize, labeled, unlabeled]
```

Comment:
I was going mad why this shit wasn't working lol.

Reply:
Yeah as obscure as it gets, I only caught it because of the VS code extension complained about it

Reply:
Yeah why isn't it an error to use non existent types :(